apps/web/screens/Grants/SearchResults/SearchResults.tsx

```diff
@@ -0,0 +1,470 @@
+        imageLayout="left"
+        breadcrumbs={
+          breadcrumbItems && (
+            <Breadcrumbs
+              items={breadcrumbItems ?? []}
+              renderLink={(link, item) => {
+                return item?.href ? (
+                  <NextLink href={item?.href} legacyBehavior>
+                    {link}
+                  </NextLink>
+                ) : (
+                  link
+                )
+              }}
+            />
+          )
+        }
+      />
+      <Box background="blue100">
+        <SidebarLayout
+          fullWidthContent={true}
+          sidebarContent={
+            <GrantsSearchResultsFilter
+              searchState={searchState}
+              onSearchUpdate={updateSearchStateValue}
+              onReset={onResetFilter}
+              tags={tags ?? []}
+              url={currentUrl}
+            />
+          }
+        >
+          {!loading && !error && (
+            <Box marginLeft={3}>
+              <Box marginBottom={3}>
+                <Text>{hitsMessage}</Text>
+              </Box>
+              {grants?.length ? (
+                <Inline space={3}>
+                  {grants?.map((grant) => {
+                    if (!grant) {
+                      return null
+                    }
+
+                    let tagVariant: TagVariant | undefined
+                    switch (grant.status) {
+                      case GrantStatus.Open:
+                        tagVariant = 'mint'
+                        break
+                      case GrantStatus.Closed:
+                        tagVariant = 'rose'
+                        break
+                      case GrantStatus.OpensSoon:
+                        tagVariant = 'purple'
+                        break
+                      default:
+                        break
+                    }
+
+                    return (
+                      <Box key={grant.id}>
+                        {grant.applicationId && (
+                          <PlazaCard
+                            eyebrow={grant.fund?.title ?? grant.name ?? ''}
+                            subEyebrow={grant.fund?.parentOrganization?.title}
+                            title={grant.name ?? ''}
+                            text={grant.description ?? ''}
+                            logo={
+                              grant.fund?.parentOrganization?.logo?.url ?? ''
+                            }
+                            logoAlt={
+                              grant.fund?.parentOrganization?.logo?.title ?? ''
+                            }
+                            tag={{
+                              label: grant.statusText ?? '',
+                              variant: tagVariant,
+                            }}
+                            cta={{
+                              label: 'Skoða nánar',
+                              variant: 'text',
+                              onClick: () => {
```

Comment:
This most often is just a link right? I think it'd be a better user experience if this would end up as an <a> tag for the user

![Screenshot 2024-11-14 at 09 31 52](https://github.com/user-attachments/assets/8a19ec4a-4a89-4b74-9d67-bd68a675d758)

Reply:
Yeah, just a link. What do you mean by "tag for the user"? For filtering or?

Reply:
<a> html tag is what I meant to write

Reply:
"<a>" html tag

Reply:
Yeah, I agree. These cards will be switched out later.